backend/src/nodes/nodes/pytorch/convert_to_onnx.py

```diff
@@ -19,17 +20,37 @@
 class ConvertTorchToONNXNode(NodeBase):
     def __init__(self):
         super().__init__()
-        self.description = """Convert a PyTorch model to ONNX."""
-        self.inputs = [ModelInput("PyTorch Model")]
-        self.outputs = [OnnxModelOutput(label="ONNX Model")]
+        self.description = """Convert a PyTorch model to ONNX.
+            Note: fp16 conversion will only work if PyTorch fp16 mode is turned on."""
+        self.inputs = [
+            ModelInput("PyTorch Model"),
+            OnnxFpDropdown(),
+        ]
+        self.outputs = [
+            OnnxModelOutput(label="ONNX Model"),
+            TextOutput(
+                "FP Mode",
+                """match Input1 {
+                    FpMode::fp32 => "fp32",
+                    FpMode::fp16 => "fp16",
+                }""",
```

Comment:
It feels like we should have a Navi function in chainner scope that does this string conversion. Right now, we already have 2 copies of this and I don't want to add a third.

Just something like this:
```
def FpMode::toString(mode: FpMode) {
  match mode {
    FpMode::fp32 => "fp32",
    FpMode::fp16 => "fp16",
  }
}

// call
FpMode::toString(Input1)
```